src/utilities/effects.ts

```diff
@@ -834,6 +855,61 @@ const effects = {
     return null;
   },
 
+  async createParcel(parcel: ParcelInsertInput, user: User | null): Promise<number | null> {
+    try {
+      if (!queryPermissions.CREATE_PARCEL(user)) {
+        throwPermissionError('create a parcel');
+      }
+
+      const data = await reqHasura<Pick<Parcel, 'id'>>(gql.CREATE_PARCEL, { parcel }, user);
+      const { createParcel } = data;
+
+      if (createParcel === null) {
+        throw Error(`Unable to create parcel "${parcel.name}"`);
+      }
+
+      const { id } = createParcel;
+      showSuccessToast('Parcel Created Successfully');
+      return id;
+    } catch (e) {
+      catchError('Parcel Create Failed', e as Error);
+      showFailureToast('Parcel Create Failed');
+      return null;
+    }
+  },
+
+  async createParcelToParameterDictionaries(
+    parcelOwner: UserId,
+    parcelToParameterDictionariesToAdd: Omit<ParcelToParameterDictionary, 'id'>[],
+    user: User | null,
+  ): Promise<ParcelToParameterDictionary[] | null> {
+    try {
+      if (!queryPermissions.UPDATE_PARCEL(user, { owner: parcelOwner })) {
```

Comment:
I think this needs to check the `CREATE_PARCEL_TO_PARAMETER_DICTIONARIES` permission